packages/gatsby-plugin-feed/src/internals.js

```diff
@@ -74,3 +76,24 @@ export const defaultOptions = {
     },
   ],
 }
+
+// https://stackoverflow.com/questions/13852852/surely-es6-must-have-a-way-to-merge-two-javascript-objects-together-what-is-it
+const isObject = item => item && typeof item === 'object' && !Array.isArray(item)
+
+export function mergeDeep(target, ...sources) {
```

Comment:
If that's preferable, cool. I don't care either way. No other reason besides this doesn't pull in a library for one function.